extensions/Webhooks/template/en/default/account/prefs/webhooks.html.tmpl

```diff
@@ -0,0 +1,169 @@
+
+<script src="[% 'js/productform.js' FILTER version FILTER html %]"></script>
+
+<script [% script_nonce FILTER none %]>
+function onSelectProduct() {
+  var component = document.getElementById('component');
+  selectProduct(document.getElementById('product'), component);
+  // selectProduct only supports __Any__ on both elements
+  // we only want it on component, so add it back in
+  try {
+    component.add(new Option('__Any__', ''), component.options[0]);
+  } catch(e) {
+    // support IE
+    component.add(new Option('__Any__', ''), 0);
+  }
+  document.getElementById('component').options[0].selected = true;
+}
+
+function onRemoveChange() {
+  var cbs = document.getElementById('webhooks_table').getElementsByTagName('input');
+  for (var i = 0, l = cbs.length; i < l; i++) {
+    if (cbs[i].checked) {
+      document.getElementById('remove').disabled = false;
+      return;
+    }
+  }
+  document.getElementById('remove').disabled = true;
+}
+
+window.onload = function() {
+  onSelectProduct();
+  onRemoveChange();
+};
+</script>
+
+<p>
+  It will be sent a POST request with the information of the [% terms.bugs %] that match with the events and filters selected to your URL.<br>
+  Documentation about webhooks is available <a href="page.cgi?id=webhooks.html">here</a>.
+</p>
+
+<table border="0" cellpadding="3" cellspacing="0">
+<tr>
+  <th align="right">Name:</th>
+  <td colspan="2"><input type="text" name="name" id="name" maxlength="64"></td>
+</tr>
+<tr>
+  <th align="right">URL:</th>
+  <td colspan="2"><input type="text" name="url" id="url" maxlength="64"></td>
+</tr>
+</table>
+<h4>Events</h4>
+<p>Select the events you want to receive.</p>
+<p>
+  <input type="checkbox" id="create_event" name="create_event" value="1">
+  <label for="create_event">Create</label>
+<br>
+  <input type="checkbox" id="change_event" name="change_event" value="1">
+  <label for="change_event">Change</label>
+</p>
+<h4>Filters</h4>
+<p>
+  To receive all components in a product, select "__Any__".
+</p>
+<table border="0" cellpadding="3" cellspacing="0">
+<tr>
+  <th>Product:</th>
+  <td>
+    <select name="add_product" id="product" onChange="onSelectProduct()">
+    [% FOREACH product IN selectable_products %]
+      <option>[% product.name FILTER html %]</option>
+    [% END %]
+    </select>
+  </td>
+</tr>
+<tr>
+  <th>Component:</th>
+  <td>
+    <select name="add_component" id="component">
+      <option value="">__Any__</option>
+    [% FOREACH product IN selectable_products %]
+      [% FOREACH component IN product.components %]
+        <option>[% component.name FILTER html %]</option>
+      [% END %]
+    [% END %]
+    </select>
+  </td>
+</tr>
+<br>
+<tr>
+  <td>&nbsp;</td>
+  <td><input type="submit" id="add_webhook" name="add_webhook" value="Add"></td>
+</tr>
+</table>
+
+<hr>
+<h3>
+  Your webhooks:
+</h3>
+
+[% IF webhooks.size %]
+
+  <table id="webhooks_table" class="standard">
+  <thead>
+    <tr>
+      <th>&nbsp;</th>
+      <th>ID</th>
```

Comment:
I added the ID thinking that the queues are named with the ID so if there's a problem and a user ask for it can said which ID has and the administrator can easily find it. Do you think could be the case or I remove it?